distributed/utils.py

```diff
@@ -949,6 +952,17 @@ def nbytes(frame, _bytes_like=(bytes, bytearray)):
             return frame.nbytes
 
 
+def PeriodicCallback(callback, callback_time, io_loop=None):
+    """
+    Wrapper around tornado.IOLoop.PeriodicCallback, for compatibility
+    with removal of the `io_loop` parameter in Tornado 5.0.
+    """
+    if tornado.version_info >= (5,):
+        return tornado.ioloop.PeriodicCallback(callback, callback_time)
+    else:
+        return tornado.ioloop.PeriodicCallback(callback, callback_time, io_loop)
```

Comment:
Do we ever actually provide an `io_loop` keyword?   If not then can we remove this function entirely?

Reply:
Well, yes, we do, in a lot of places :-)

Reply:
It seems odd to me that we are allowed to ignore those imports in Tornado >= 5.0 but need them in <= 5.0.

Reply:
That's because we fixed `PeriodicCallback` so that the IO loop is looked up when its `start` method is called. Before 5.0 the IO loop is set in the constructor, and we call it from another thread.